src/dc_chat.rs

```diff
@@ -1740,21 +1744,19 @@ pub unsafe fn dc_remove_contact_from_chat(
                         (*msg).type_0 = DC_MSG_TEXT;
                         if (*contact).id == 1 as libc::c_uint {
                             dc_set_group_explicitly_left(context, (*chat).grpid);
-                            (*msg).text = dc_stock_system_msg(
-                                context,
-                                19,
-                                0 as *const libc::c_char,
-                                0 as *const libc::c_char,
-                                1 as uint32_t,
-                            )
+                            (*msg).text = to_cstring(context.stock_system_msg(
+                                StockMessage::MsgGroupLeft,
+                                "",
+                                "",
+                                DC_CONTACT_ID_SELF as u32,
+                            ));
                         } else {
-                            (*msg).text = dc_stock_system_msg(
-                                context,
-                                18,
-                                (*contact).addr,
-                                0 as *const libc::c_char,
-                                1 as uint32_t,
-                            )
+                            (*msg).text = to_cstring(context.stock_system_msg(
+                                StockMessage::MsgDelMember,
+                                as_str((*contact).addr),
+                                "",
+                                DC_CONTACT_ID_SELF as u32,
```

Comment:
maybe declare DC_CONTACT_ID_SELF as u32?

Reply:
It's used in the callback function as well which expects usize.  I think that'd be for a later refactor which improves callback handling further.